R/module-repository.R

```diff
@@ -364,39 +382,48 @@ setMethod(
 setMethod(
   "downloadModule",
   signature = c(name = "character", path = "missing", version = "missing",
-                repo = "missing", data = "missing", quiet = "missing", quickCheck = "ANY"),
+                repo = "missing", data = "missing", quiet = "missing", quickCheck = "ANY",
+                overwrite = "missing"),
   definition = function(name, quickCheck) {
     files <- downloadModule(name, path = getOption("spades.modulePath"),
                             version = NA_character_,
                             repo = getOption("spades.moduleRepo"),
                             data = FALSE, quiet = FALSE,
-                            quickCheck = quickCheck)
+                            quickCheck = quickCheck, overwrite = FALSE)
     return(invisible(files))
 })
 
 #' @rdname downloadModule
 setMethod(
   "downloadModule",
   signature = c(name = "character", path = "ANY", version = "ANY",
-                repo = "ANY", data = "ANY", quiet = "ANY", quickCheck = "ANY"),
-  definition = function(name, path, version, repo, data, quiet, quickCheck) {
+                repo = "ANY", data = "ANY", quiet = "ANY", quickCheck = "ANY",
+                overwrite = "ANY"),
+  definition = function(name, path, version, repo, data, quiet, quickCheck,
+                        overwrite) {
     if (missing(path)) path <- getOption("spades.modulePath")
     if (missing(version)) version <- NA_character_
     if (missing(repo)) repo <- getOption("spades.moduleRepo")
     if (missing(data)) data <- FALSE
     if (missing(quiet)) quiet <- FALSE
+    if (missing(overwrite)) overwrite <- FALSE
```

Comment:
note that default should already be set in the generic

Reply:
Ok, I deleted it